pyproject.toml

```diff
@@ -7,47 +11,52 @@ readme = "README.md"
 license = {file = "LICENSE"}
 dependencies = ['typing-extensions']
 classifiers = [
-    "Programming Language :: Rust",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Python :: Implementation :: PyPy",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: POSIX :: Linux",
-    "Operating System :: MacOS",
-    "Operating System :: Microsoft :: Windows",
-    "Topic :: System :: Networking"
+  "Programming Language :: Rust",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: MacOS",
+  "Operating System :: Microsoft :: Windows",
+  "Topic :: System :: Networking"
 ]
 authors = [
-    { name = "Samuel Yvon", email = "samuelyvon9@gmail.com" }
+  {name = "Samuel Yvon", email = "samuelyvon9@gmail.com"}
 ]
 maintainers = [
-    { name = "Samuel Yvon", email = "samuelyvon9@gmail.com" }
+  {name = "Samuel Yvon", email = "samuelyvon9@gmail.com"}
 ]
 
 [project.urls]
-homepage = "https://github.com/SamuelYvon/netifaces-2"
-issues = "https://github.com/SamuelYvon/netifaces-2/issues"
+Homepage = "https://github.com/SamuelYvon/netifaces-2"
+Issues = "https://github.com/SamuelYvon/netifaces-2/issues"
 
 [project.optional-dependencies]
 dev = [
-    "mypy >= 0.981",
-    "black >= 22.10.0",
-    "isort >= 5.10.1",
+  "mypy >= 0.991",
+  "black >= 22.12.0",
+  "ruff >= 0.0.238"
 ]
 
-[build-system]
-requires = ["maturin>=0.14,<0.15"]
-build-backend = "maturin"
+[tool.black]
+target-version = ["py311", "py310", "py39", "py38", "py37", "py36"]
 
 [tool.maturin]
 bindings = "pyo3"
 
-[tool.isort]
-profile = "black"
+[tool.mypy]
+python_version = "3.6"
+ignore_missing_imports = true
```

Comment:
Unsure how you setup the python env, but I use `venv` here. `mypy` does not like the content of the the `venv` folder so I'd either add this line here:
```yaml
exclude = ['venv/.*/*\.py$']
```
or switch to a `.mypy.ini` file

Reply:
I have virtualenvs in different directory than the projects. Same as if you used pipx. Don't know about pipenv.

Reply:
Sure sure, it was not removed intentionally.  
I'm afk today - will fix it all tomorrow.